.NET/Microsoft.Recognizers.Text.Number/Parsers/BaseNumberParser.cs

```diff
@@ -597,6 +598,24 @@ protected double GetDigitalValue(string digitsStr, double power)
                 {
                     isNegative = true;
                 }
+
+                // Used for Hindi to resolve Devanagari numerals, C# char.isDigit accouts for Devanagari Numbers

```

Comment:
Please never break a if/else with comments in the middle. Comments can always go inside the clause.
However, such changes shouldn't be added to base code if Hindi-specific. Hindi could extend BaseNumberParser and override specific methods. Or new overridable methods can be created and Hindi config implements them differently from the default config.
Urudu will probably reuse these Hindi-specific changes, right?

Reply:
Noted. 
The majority of Indian languages e.g. Bengali, Urdu, Gujarati, Marathi, Punjabi and other Indo-Aryan and Dravidian languages have the same behavior. 

We will extend BaseNumberParser for Hindi for now.